efficientdet/anchors.py

```diff
@@ -350,25 +352,24 @@ def _generate_detections_tf(cls_outputs,
   scores = tf.math.sigmoid(cls_outputs)
   # apply bounding box regression to anchors
   boxes = decode_box_outputs_tf(box_outputs, anchor_boxes)
-
-  top_detection_idx, scores = tf.image.non_max_suppression_with_scores(
-      boxes,
-      scores,
-      max_boxes_to_draw,
+  top_detection_idx, scores, _ = tf.raw_ops.NonMaxSuppressionV5(
+      boxes=boxes,
```

Comment:
Is this op faster than tf.image.non_max_suppression_with_scores?

Reply:
```tf.image.non_max_suppression_with_scores``` doesn't support float16.
It's a bad wrapper.